### DISCUSSION TITLE
Ord wallet balance command - Complete indexing of the blocks stops.

### DISCUSSION THREAD

**Discussion by cideight** [ordinals/ord#1921] [Q&A] [answered]
After command : ./ord wallet balance 
![image](https://user-images.githubusercontent.com/12185377/224534527-ddff352f-c06f-4ba5-ae49-b88d10e406c3.png)

but i have 100% indexing score and current blockheight on my fullnode env.
![image](https://user-images.githubusercontent.com/12185377/224551331-177c02e6-1cca-4e61-8ea8-ced3674e9776.png)

I transfered some satoshis to my ord wallet as test and i see the correct amount.
{
  "walletname": "ord",
  "walletversion": 169900,
  "format": "sqlite",
  "balance": 0.00089800,
  "unconfirmed_balance": 0.00000000,
  "immature_balance": 0.00000000,
  "txcount": 1,
  "keypoolsize": 1000,
  "keypoolsize_hd_internal": 1000,
  "paytxfee": 0.00000000,
  "private_keys_enabled": true,
  "avoid_reuse": false,
  "scanning": false,
  "descriptors": true,
  "external_signer": false
.....

Ord wallet is newest version, compiled from scratch, same for BTC full node with txindex=1 activ
Reindex is not the solution, but thanks for every hint or solution...

**Comment by cideight** [comment, marked as answer]
It works now. after changed from a raspi 3 to a Intel NUC hardware the blocks going flawless trough. So it gives a other option too. You need only find the red index db on the internet. Then you can replace the existing with this up to date one.

**Comment by GaloisField2718** [comment]
Hello,

I have an issue with the output balance it told me : 30,817 but in reality there are 125,542 sats.
My opinion was that balance takes only some address. But, when I take a look with Sparrow, there aren't any address with 30,817 neither address's balances such that balance sum are equal to 30,817.

Do you think that I must modify subcommands/balance.rs ?

I tried this code : 
```rs
let client = options.bitcoin_rpc_client_for_wallet_command(true)?;
  for (outpoint, amount) in client.call("getbalance", &["bech32".into()])? {
    if !inscription_outputs.contains(&outpoint) {
      balance += amount
    }
  }
``` 
but it doesn't work. 

Do you have any idea about this ?
